etc/profile-m-z/makepkg.profile

```diff
@@ -1,4 +1,5 @@
 # Firejail profile for makepkg
+# Description: An utility to automate the building of Arch Linux packages
```

Comment:
```suggestion
# Description: A utility to automate the building of Arch Linux packages
```

Grammar: "utility" starts with a consonant sound ("you"), so it's "a utility".

Related:

* <https://github.com/netblue30/firejail/pull/5290#discussion_r936162181>